.sass-lint.yml

```diff
@@ -23,6 +23,7 @@ rules:
         - border-top
         - border-bottom
         - cursor
+        - style
```

Comment:
@nschonni @LaurentGoderre Are you guys ok with what I did here? Does ``- style`` seem too generic?

In ``src/polyfills/details/_base.scss``, I'm currently using nested style properties within the ``list`` namespace. Originally I tried setting this line to ``- list-style``, but got this error: ``Duplicate properties are not allowed within a block  no-duplicate-properties``.

So I think I only have 2 options for what I'm trying to achieve:
1. What I'm already doing (``- style`` on this line and nested properties in the ``list`` name space)
2. ``list-style`` on this line and two consecutive ``list-style`` properties.

Thoughts?